components/AlertModal.tsx

```diff
@@ -0,0 +1,60 @@
+import { Button } from '@chakra-ui/button';
+import { WarningTwoIcon } from '@chakra-ui/icons';
+import { Flex, Text } from '@chakra-ui/layout';
+import {
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+} from '@chakra-ui/modal';
+import { Address } from 'abitype';
+
+type AlertModalProps = {
+  isOpen: boolean;
+  onClose: () => void;
+  onClick: () => void;
+  address: Address;
+  description: string;
+  children?: React.ReactNode;
+};
+
+const AlertModal = (props: AlertModalProps) => {
```

Comment:
Should this be changed to ErrorModal? My thought was we could support different levels of alerts like info or warning.

Reply:
I'm indifferent here, will defer to you and @wildmolasses

Reply:
sounds good for this PR, we can refactor later

Reply:
I ended up renaming to WarningModal yesterday